File: backend/api_server/pkg/configs/log_config.go
package configs

import (
	"os"
	"strconv"
)

type LogConfigResult struct {
	MaxSize int
	MaxBackups int 
	MaxAge int 
	Compress bool
	LocalTime bool
}

func LogConfig() LogConfigResult {
	maxSize := os.Getenv("LOG_MAX_SIZE")
	maxBackups := os.Getenv("LOG_MAX_BACKUPS")
	maxAge := os.Getenv("LOG_MAX_AGE")
	compress := os.Getenv("LOG_COMPRESS") == "true"
	localTime := os.Getenv("LOG_LOCAL_TIME") == "true"

	maxSizeInt, err := strconv.ParseInt(maxSize, 10, 64)
	if err != nil {
		maxSizeInt = 10
	}

	maxBackupsInt, err := strconv.ParseInt(maxBackups, 10, 64)
	if err != nil {
		maxBackupsInt = 5
	}

	maxAgeInt, err := strconv.ParseInt(maxAge, 10, 64)
	if err != nil {
		maxAgeInt = 30
	}

	return LogConfigResult{
		MaxSize: int(maxSizeInt), 
		MaxBackups: int(maxBackupsInt),
		MaxAge: int(maxAgeInt),
		Compress: compress,
		LocalTime: localTime,
	}
}

File: backend/ai_server/ai_server/pkg/configs/log_config.py
import os
from dataclasses import dataclass

@dataclass
class LogConfigResult:
    maxSize: int 
    maxBackups: int
    maxAge: int
    compress: bool
    localTime: bool

def get_log_config() -> LogConfigResult:
    return LogConfigResult(
        maxSize=int(os.getenv("LOG_MAX_SIZE", 10)),
        maxBackups=int(os.getenv("LOG_MAX_BACKUPS", 5)),
        maxAge=int(os.getenv("LOG_MAX_AGE", 30)),
        compress=bool(os.getenv("LOG_COMPRESS", True)),
        localTime=bool(os.getenv("LOG_LOCAL_TIME", True))
    )